### DISCUSSION TITLE
connection

### DISCUSSION THREAD

**Discussion by mo5tarasran** [MohamedKVIP/MK-PUBG-Mobile-Tool#11] [Q&A]
البرنامج بيخرج تلقائي معايا من قبل حتي ميعمل connect ب gameloop

**Comment by MohamedKVIP** [comment]
مرحبًا @mo5tarasran ،

شكرًا على إبلاغك. يبدو أنك تواجه مشكلة إغلاق مفاجئ قبل أن تبدأ عملية الاتصال. نحن على علم بهذه المشكلة ونعمل على إصلاحها في التحديث القادم.

في هذه الأثناء، قد ترغب في تجربة إصدار AIO الخاص بنا، والذي يستخدم طريقة اتصال مختلفة. يمكنك العثور عليه على خادم Discord الخاص بنا: [الديسكورد](https://discord.gg/PDPJM6e6PC)

نقدر سعة صدرك.
